scss/_regions/_container.scss

```diff
@@ -59,6 +60,8 @@
       }
 
       &.-half {
+        line-height: $unitless-line-height; // reset to "base" line height for smaller columns
```

Comment:
Yeah, I think it'd be worth thinking a bit more about the class naming & rules on when we use the more "expanded" line spacing. I think this should be a "drop in" solution for places where @lkpttn wants this change to apply, but it could make sense to make this be a new modifier or pattern instead...

Context from Luke's design spec:

> We’ll also be adjusting line-height based on container width. The large size (507px+) will default to `28.125px` and anything smaller with use `26px`. This can be considered an experimental change and could be reverted if the impact is deemed too minor. We’ve seen this become a standard practice with companies that focus on longform text, like Medium, NYT, etc